variables.tf

```diff
@@ -132,6 +132,18 @@ variable "artifact_location" {
   description = "Location of artifact. Applies only for artifact of type S3"
 }
 
+variable "secondary_artifact_location" {
+  type        = string
+  default     = ""
+  description = "Location of secondary artifact. Must be an S3 reference"
+}
+
+variable "secondary_artifact_identifier" {
+  type        = string
+  default     = "static"
```

Comment:
Will anyone ever use the default of `static` or will this always be overridden? If it's expected that `static` will never be used then I think we should set to `null` as well.